portfolio/src/main/webapp/script.js

```diff
@@ -34,15 +34,37 @@ async function getName() {
 }
 
 async function getComment() {
+  const visibility = document.getElementById("hideorshow");
+  const hidelogin = document.getElementById("hidelogin");
+  const showlogout = document.getElementById("showlogout");
+  const login = await fetch('/login');
+  const loginstatus = await login.text();
+  const loginName = loginstatus.split("<p");
+  if(loginName[0].includes("please sign in to leave a comment.")){
```

Comment:
Like I commented below, I think this would be easier to reason about if you returned JSON from the login servlet.